freqtrade/exchange/exchange.py

```diff
@@ -386,6 +386,13 @@ def validate_ordertypes(self, order_types: Dict) -> None:
                 f'On exchange stoploss is not supported for {self.name}.'
             )
 
+        # Limit price threshold: As limit price should always be below stop-price
+        # Used for limit stoplosses on exchange
+        limit_price_pct = order_types.get('stoploss_on_exchange_limit_ratio', 0.99)
+        if limit_price_pct >= 1.0 or limit_price_pct <= 0.0:
+            raise OperationalException(
+                "stoploss_on_exchange_limit_ratio should be < 1.0 and > 0.0")
+
```

Comment:
I don't think this is exchange specific - so mabye adding it to the validation in constants.py would suffice? (you can add min / max values there ...)

Reply:
with this addition to constants, the following validation comes up:

```
                'stoploss_on_exchange_limit_ratio': {'type': 'number', 'minimum': 0.0,
                                                     'maximum': 1.0}
```

```
2020-07-13 07:14:22,436 - freqtrade.configuration.config_validation - INFO - Validating configuration ...
2020-07-13 07:14:22,437 - freqtrade.configuration.config_validation - CRITICAL - Invalid configuration. See config.json.example. Reason: -1.0 is less than the minimum of 0.0

Failed validating 'minimum' in schema['properties']['order_types']['properties']['stoploss_on_exchange_limit_ratio']:
    {'maximum': 1.0, 'minimum': 0.0, 'type': 'number'}

On instance['order_types']['stoploss_on_exchange_limit_ratio']:
    -1.0
2020-07-13 07:14:22,439 - freqtrade.commands.trade_commands - ERROR - -1.0 is less than the minimum of 0.0

```

This is followed by another, "not so nice" stacktrace - but i'd rather take that in a seperate PR.

@hroff-1902 if you agree, i'll commit this to this branch and remove the above, explicit test (as that cannot be hit anymore ...).

Reply:
yes, please, let's complete it...

I remember I was wanted to revert some code in the commits above, and cannot recall in the meantime how I wanted to complete this PR a month ago, following your comments above.

Reply:
for sure - let me know what you think - it's the last (2) commits which remove this test in place of testing it as part of the configuration.